### PR TITLE
Update the reconnecting RemoteUFD according to figma

### PR DESCRIPTION
**Pull request by PorterNan**
# What

image

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/components/styles/StreamMedia.styles.ts [hunk 1]
```diff
@@ -76,6 +59,21 @@ export const loadSpinnerStyles: ISpinnerStyles = {
   }
 };
 
+/**
+ * @private
+ */
+export const reconnectSpinnerStyles = (theme: Theme): ISpinnerStyles => ({
+  ...loadSpinnerStyles,
+  label: {
+    color: theme.palette.white
```
[the remark this answers is not in the document]
Comment (JamesBurnside) [reply packages/react-components/src/components/styles/StreamMedia.styles.ts:68]: That's a fixed background you can choose for the theme generator and not the background behind the text at runtime

[the remark this answers is not in the document]
Comment (JamesBurnside) [reply packages/react-components/src/components/styles/StreamMedia.styles.ts:68]: I would test locally but I don't know how to get into this remote reconnecting state 😅

[the remark this answers is not in the document]
Comment (PorterNan) [reply packages/react-components/src/components/styles/StreamMedia.styles.ts:68]: You will need to use mobile phone then going into flight mode, I will also take a try soon today!

[the remark this answers is not in the document]
Comment (PorterNan) [reply packages/react-components/src/components/styles/StreamMedia.styles.ts]: Done! Make the color to white no matter what mode it is, thanks for catching this!

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**